src/seedu/addressbook/commands/SortNameCommand.java

```diff
@@ -0,0 +1,42 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.Comparator;
+import java.util.List;
+
+/**
+ * Sorts the address book according by names in alphabetical order.
+ */
+
+public class SortNameCommand extends Command{
+
+    public static final String COMMAND_WORD = "sortname";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts all data of persons in address book "
+            + "in alphabetical order according to their name.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SORT_ACKNOWEDGEMENT = "Address Book has been sorted.";
+
+    public static class nameComparator implements Comparator<Person> {
```

Comment:
I would suggest not to have this logic in SortNameCommand class. The responsibility of this class is to handle all the necessary information related to sort command and communicate with addressbook when you require to sort (execute). One good option will be, moving it to  **UniquePersonList**. Addressbook will communicate with UniquePersonList to sort similar to add, remove etc. This is a good example of practicing **Single Responsibility Principle**.